packages/core/src/components/Tabs/index.tsx

```diff
@@ -46,6 +46,18 @@ export class Tabs extends React.Component<Props & WithStylesProps, State> {
     selectedKey: this.getDefaultSelectedKey(),
   };
 
+  boundPopstateHandler = this.handlePopstate.bind(this);
+
+  componentDidMount() {
+    if (this.props.persistWithHash) {
+      window.addEventListener('popstate', this.boundPopstateHandler);
```

Comment:
in lunar, we use `document` instead of `window` for event listeners. please update!